configurable-http-proxy.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: configurable-http-proxy
+  version: 4.6.0
+  epoch: 0
+  description: "HTTP parser written against llparse"
+  copyright:
+    - license: BSD 3-Clause
+  dependencies:
+    runtime:
+      - nodejs-16
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - python3
+      - nodejs-16
+      - npm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 789f171a5d4eca622e6c344b2cecdbc7aeba4e46
+      repository: https://github.com/jupyterhub/configurable-http-proxy
+      tag: ${{package.version}}
+
+  - uses: patch
+    with:
+      patches: 0001-fix-GHSA-72xf-g2v4-qvf3.patch
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/srv/${{package.name}}
+      cp chp-docker-entrypoint package.json index.js ${{targets.destdir}}/srv/${{package.name}}/
+      cp -r bin/ doc/ lib/ test/ ${{targets.destdir}}/srv/${{package.name}}/
+
+  - working-directory: ${{targets.destdir}}/srv/${{package.name}}
+    pipeline:
+      - runs: |
+          npm install --package-lock-only
+          npm ci --production
+          npm audit fix --package-lock-only --legacy-peer-deps || true
```

Comment:
Why `|| true`?

Reply:
Hmmm, no particular reason for this, copy-paste issue probably. But i think its a nit.